tools/config/schema_lib.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Mbed Library Schema",
+  "description": "Configuration file for an mbed library",
+  "type": "object",
+  "$id": "http://example.com/root.json",
+  "definitions": {
```

Comment:
These definitions look almost the same. Could they be moved to another file, and imported?

Reply:
Yeah, that should be possible. I'll have a look at separating out the common parts of them, I don't think there are any differences between app and lib config parameter handling that needs to be handled in the validation.